app/views/account/PaymentsView.coffee

```diff
@@ -2,11 +2,19 @@ RootView = require 'views/core/RootView'
 template = require 'templates/account/payments-view'
 CocoCollection = require 'collections/CocoCollection'
 Payments = require 'collections/Payments'
+Prepaids = require 'collections/Prepaids'
 
 module.exports = class PaymentsView extends RootView
   id: "payments-view"
   template: template
 
   initialize: ->
     @payments = new Payments()
-    @supermodel.trackRequest(@payments.fetch({cache: false}))
+    @supermodel.trackRequest @payments.fetchByCreator(me.id)
+    @prepaids = new Prepaids()
+    @supermodel.trackRequest @prepaids.fetchByCreator(me.id)
+
+  onLoaded: ->
+    @prepaidMap = _.zipObject(_.map(@prepaids.models, (m) => m.id), @prepaids.models)
```

Comment:
Any particular reason not to use `view.prepaids.get(id)` in the template instead of this?